druid/examples/image.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 The xi-editor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This example shows how to draw an SVG.
+//!
+//! Requires the non-default "svg" feature to be enabled:
+//! `cargo run --example svg --features "svg"`
+//!
+
+#[cfg(not(feature = "images"))]
+fn main() {
+    eprintln!("This examples requires the \"images\" feature to be enabled:");
+    eprintln!("cargo run --example image --features \"images\"");
+}
+
+#[cfg(feature = "images")]
+use std::str::FromStr;
+
+#[cfg(feature = "images")]
+use druid::{
+    widget::{FillStrat, Flex, Image, ImageData, WidgetExt},
+    AppLauncher, Widget, WindowDesc,
+};
+
+#[cfg(feature = "images")]
+fn main() {
+    let main_window = WindowDesc::new(ui_builder);
+    let data = 0_u32;
+    AppLauncher::with_window(main_window)
+        .use_simple_logger()
+        .launch(data)
+        .expect("launch failed");
+}
+
+#[cfg(feature = "images")]
+fn ui_builder() -> impl Widget<u32> {
+    let png_data = ImageData::from_str("examples/pngexample.png").unwrap();
+
+    let mut col = Flex::column();
+
+    col.add_child(
+        Image::new(png_data.clone(), FillStrat::Contain)
+            .fix_width(100.0)
+            .center(),
+        1.0,
+    );
+    let otherimage = Image::new(png_data, FillStrat::FitWidth);
+    //Image::set_fill(&mut otherimage, FillStrat::ScaleDown);
```

Comment:
cleanup

Reply:
I left this as a hint to others as to things that they can do, if i add comment to that effect would that be ok or shall i just remove?